deploy/deploy.go

```diff
@@ -901,13 +901,22 @@ func (i *IxiaTGSpec) Deploy(ctx context.Context) error {
 		return err
 	}
 	if i.ConfigMap == nil {
-		path := filepath.Join(i.ManifestDir, "ixia-configmap.yaml")
-		if _, err := osStat(path); err != nil {
-			log.Warnf("ixia controller deployed without configmap, before creating a topology with ixia-c be sure to create a configmap following https://github.com/open-traffic-generator/ixia-c-operator#ixia-c-operator and apply it using 'kubectl apply -f ixia-configmap.yaml'")
+		var cmPath string
+		for _, cmName := range []string{"ixiatg-version.yaml", "ixiatg-configmap.yaml", "ixia-configmap.yaml"} {
+			path := filepath.Join(i.ManifestDir, cmName)
+			if _, err := osStat(path); err != nil {
```

Comment:
more useful to log which filepath was chosen, if none are found then the warning below is already enough